web-starter/templates/capistrano/drupal8.rb

```diff
@@ -0,0 +1,86 @@
+# Use version of Drush supplied via Composer
+SSHKit.config.command_map[:drush] = "../vendor/drush/drush/drush"
+  
+# Revert the database when a rollback occurs
+Rake::Task["deploy:rollback_release_path"].enhance do
+  invoke "drupal8:revert_database"
+end
+
+# Backup the database when publishing a new release
+Rake::Task["deploy:publishing"].enhance ["drupal8:dbbackup"]
+
+# Copy drush aliases after linking the new release
+Rake::Task["deploy:symlink:release"].enhance ["drush:initialize"]
+
+# After publication run updates
+Rake::Task["deploy:published"].enhance do 
+  Rake::Task["drush:update"].invoke
+end
+
+namespace :drupal8 do
+  desc "Install Drupal"
+  task :install do
+    invoke 'drush:siteinstall'
+  end
+  
+  desc "Copy Drupal and web server configuration files"
+  task :settings do
+    on roles(:app) do
+      fetch(:site_folder).each do |folder|
+        # Find and link settings.php
+        if test " [ -e #{current_path}/#{fetch(:app_webroot, 'public')}/sites/#{folder}/settings.php ]"
+          execute :rm, "-f", "#{current_path}/#{fetch(:app_webroot, 'public')}/sites/#{folder}/settings.php"
+        end
+        execute :ln, '-s', "#{current_path}/#{fetch(:app_webroot, 'public')}/sites/#{folder}/settings.#{fetch(:stage)}.php", "#{current_path}/#{fetch(:app_webroot, 'public')}/sites/#{folder}/settings.php"
+
+        # Find and link services.yml
+        if test " [ -e #{current_path}/#{fetch(:app_webroot, 'public')}/sites/#{folder}/services.yml ]"
+          execute :rm, "-f", "#{current_path}/#{fetch(:app_webroot, 'public')}/sites/#{folder}/services.yml"
+        end
+        execute :ln, '-s', "#{current_path}/#{fetch(:app_webroot, 'public')}/sites/#{folder}/services.#{fetch(:stage)}.yml", "#{current_path}/#{fetch(:app_webroot, 'public')}/sites/#{folder}/services.yml"
```

Comment:
We may not always have a services.yml file -- it's not required.

Reply:
Thanks. Resolved by #10.